docs/iris/src/whatsnew/1.9.rst

```diff
@@ -0,0 +1,115 @@
+What's New in Iris 1.9
+**********************
```

Comment:
The [Sphinx docs](http://sphinx-doc.org/rest.html#sections) say "* with overline, for chapters", so I guess this needs an overline? Though it might be configurable in some way that I don't know, since the docs mention it.